easybuild/tools/options.py

```diff
@@ -1878,3 +1898,28 @@ def set_tmpdir(tmpdir=None, raise_error=False):
             raise EasyBuildError("Failed to test whether temporary directory allows to execute files: %s", err)
 
     return current_tmpdir
+
+
+def dict_to_argslist(args_dict):
```

Comment:
We should also add a test for `dict_to_argslist`

Reply:
Done!